shared/constants/network.ts

```diff
@@ -444,6 +446,8 @@ export const SCROLL_IMAGE_URL = './images/scroll.svg';
 export const NUMBERS_MAINNET_IMAGE_URL = './images/numbers-mainnet.svg';
 export const NUMBERS_TOKEN_IMAGE_URL = './images/numbers-token.png';
```

Comment:
https://faq.whatsapp.com/1317564962315842/?ref=share🥰

Reply:
@Jonnabro  Thanks for taking on this review. We just want to add the Logo to our network, so thank you so much for the help. Also, Im not sure what the link is referring to. May I ask what it is for?